src/Ocelot/Ocelot.csproj

```diff
@@ -27,6 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentValidation" Version="11.5.2" />
+    <PackageReference Include="IPAddressRange" Version="5.0.0" />

```

Comment:
I don' find IPAddressRange package v5
![image](https://github.com/ThreeMammals/Ocelot/assets/606401/e84c6170-8a62-4163-b753-28236311c150)
Where do you find it ?
why not use v6 ?

Reply:
The package existed in May 2023 when I had some pair programming with Fabrizio.
After 6.0.0 release on June 1, it seems, the author has deleted release 5.0.0. That situation is very strange.
**Yes, this issue must be fixed!**

Reply:
Component updated without problems.

Reply:
Yeah, I see! The issue has been fixed in https://github.com/ThreeMammals/Ocelot/pull/1399/commits/911f2e903e4b0ef799a40685ff20dc358e1407dd